microsoftgraph/microsoft-graph-explorer-v4
# Enhancement: Varying response heights

## Overview
Closes #2703 

Brief description of what this PR does, and why it is needed.

### Demo

Optional. Screenshots, `curl` examples, etc.

### Notes

Optional. Ancillary topics, caveats, alternative strategies that didn't work out, anything else.

## Testing Instructions

* How to test this PR
Click on the Response tabs
Notice that the heights remain consistent through the different tabs
Zoom in and out to get into mobile screen. Notice that the app is still responsive
